src_py/midi.py

```diff
@@ -68,8 +68,7 @@ def _module_init(state=None):
         _module_init.value
     except AttributeError:
         return False
-    else:
```

Comment:
Again, I think this should be in a separate PR. It's completely unrelated to `_sdl2.Window`